### ISSUE TITLE
Handling of Multiple Words

### ISSUE THREAD

**Issue by nv78** [Pandora-Intelligence/concise-concepts#34]
Seems like concise concepts, while great, isn't able to assign entities to "multiple words". For instance, it might pick up:
Mashed -> ENT
Potato -> ENT

But not "Mashed Potato" -> ENT

Is there any way that we can solve for this?

**Comment by davidberenstein1957** [comment]
I agree. @nv78 I need to look into compound chains to handle this properly. Have been playing around with this but will probably be covered during my vacation in April:)

**Comment by nv78** [comment]
Perfect, reach out at vidranatan@gmail.com if you need someone to test!

**Comment by DSoudis** [comment]
This would be a killer feature! 🔥🔥🔥

**Comment by nv78** [comment]
Agreed - we've had this issue for a while! If there is a solution for multi word entity recognizers, let me know.

**Comment by DSoudis** [comment]
It seems to be possible when using Prodigy (from Explosion AI).

Check the video below:
https://m.youtube.com/watch?time_continue=324&v=59BKHO_xBPA&embeds_euri=https%3A%2F%2Fprodi.gy%2F&feature=emb_title

The seed entities are single words (like broccoli) but the suggested entities include feta cheese and tomato soup
